pyrefly/lib/test/class_super.rs

```diff
@@ -113,6 +113,36 @@ class C:
     "#,
 );
 
+testcase!(
+    test_super_protocol_stub_method,
```

Comment:
Similar to my other rename request - could we name this `test_super_protocol_unimplemented_method`?